public/static/js/spawned_hosts.js

```diff
@@ -202,6 +217,29 @@ mciModule.controller('SpawnedHostsCtrl', ['$scope','$window', '$timeout', 'mciSp
       );
     };
 
+    $scope.updateInstanceType = function() {
+      // Do nothing if no instance type selected
+      if ($scope.selectedInstanceType.length == 0) {
```

Comment:
I would do this check as !$scope.selectedInstanceType, since this will hit an exception if it's undefined

Reply:
Fixed!